snuba/query/columns.py

```diff
@@ -43,12 +45,40 @@ def column_expr(dataset, column_name, query: Query, parsing_context: ParsingCont
         return escape_literal(column_name[1:-1])
     else:
         expr = dataset.column_expr(column_name, query, parsing_context)
-
     if aggregate:
         expr = function_expr(aggregate, expr)
 
-    alias = escape_alias(alias or column_name)
-    return alias_expr(expr, alias, parsing_context)
+    strip_negation = state.get_config('process_alias_without_neg', 0)
```

Comment:
This is a precution during rollout since here there can be a vast number of corner cases. If everything works well I will remove the flag.